contracts/BridgeTokens.sol

```diff
@@ -12,7 +12,7 @@ contract BridgeTokens is Ownable, IBridgeTokens {
 
     /// @dev Constructor function
     /// @param _supportedTokens An array of addresses representing the supported tokens
-    constructor(address[] memory _supportedTokens) Ownable() {
+    constructor(address[] memory _supportedTokens) Ownable(msg.sender) {
```

Comment:
why this change?

Reply:
Using OZ V5 library now to support the new OZ upgrades library. There are a few minor interface changes that were needed like this.